### ISSUE TITLE
ImportError Could not import sentence_transformers python package. Please install it with `pip install sentence-transformers`.

### ISSUE THREAD

**Issue by happy15** [kreneskyp/ix#466]
Hi, I bootstrapped ix with the docker solution and tried to do build a RAG agent with huggingface embedding, which needs sentence-transformers the python lib.

But it seems that sentence-transformers is not included in the docker image and I encountered reply with "ImportError
Could not import sentence_transformers python package. Please install it with `pip install sentence-transformers`."

Do I miss anything here?

Below is the flow under construction:

<img width="1190" alt="image" src="https://github.com/kreneskyp/ix/assets/983570/a6a130b0-be72-4eba-a02f-bc47013a2f18">

**Comment by saahil1801** [comment]
Hello @happy15 , yes you got that right , we need to install sentence-transformers  , a permanent solution is to modify your Dockerfile to include the installation of sentence-transformers. This way, every time you build your Docker image, the sentence-transformers library will be included. 
RUN pip install sentence-transformers
Then, you can rebuild your Docker image and restart your container. Also you can verify the same using print(sentence_transformers.__version__)

**Comment by happy15** [comment]
ok, thanks for the reply, will try.

**Comment by kreneskyp** [comment]
Yes, assuming you're building from source add the library to `requirements.txt`.  It should be included in the image when re-built.
